doc/nrf/links.txt

```diff
@@ -117,6 +117,14 @@
 .. _`Matter SDK tagged as v1.0.0`: https://github.com/project-chip/connectedhomeip/releases/tag/v1.0.0
 .. _`Testing with Apple Devices`: https://github.com/project-chip/connectedhomeip/blob/master/docs/guides/darwin.md
 .. _`Matter factory data Kconfig options`: https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/kconfig/index.html#!CHIP_FACTORY_DATA
+.. _`Bluetooth LE Arbiter's header file`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/master/src/platform/Zephyr/BLEAdvertisingArbiter.h
+.. _`Bluetooth LE Arbiter's Request struct`: https://github.com/nrfconnect/sdk-connectedhomeip/blob/master/src/platform/Zephyr/BLEAdvertisingArbiter.h#L52
```

Comment:
I'm afraid it would be hard to maintain it in the future (i.e. line number will definitely change). We can fix it reactively or, may be we should just link to `BLEAdvertisingArbiter.h` file?

Reply:
Maybe we can just insert a name of that struct here?